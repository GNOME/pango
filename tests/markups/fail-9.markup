<span font="Sans Italic 12"
      foreground="#00ff00"
      background="#f0f0f0"
      underline="double"
      underline_color="red"
      strikethrough="x"
      strikethrough_color="blue"
      fallback="false"
      lang="de"
      gravity="south"
      gravity_hint="strong">Lots of attrs</span>
