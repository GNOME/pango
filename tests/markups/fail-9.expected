ERROR: Value of 'strikethrough' attribute on <span> tag line 11 should have one of 'true/yes/t/y' or 'false/no/f/n': 'x' is not valid